### PR TITLE
Corrected RegEx's use of alternation

### PR DESCRIPTION
**Pull request by gsvarovsky**
- not sure about the correct use of the pyproject.toml `version`

### CODE REVIEW

File: tests/test_no_todo.py [hunk 1]
```diff
@@ -13,12 +13,51 @@ def test_no_todo(monkeypatch: pytest.MonkeyPatch) -> None:
...
+def test_has_todo(monkeypatch: pytest.MonkeyPatch) -> None:
+    monkeypatch.setattr("sys.argv", ["no_todos.py", "tmp.py"])
+
+    with TemporaryDirectory() as tmp_dir:
+        os.chdir(tmp_dir)
+
+        with open("tmp.py", "w") as file:
+            print("print('hello world') # TODO: update", file=file)
+
+        with pytest.raises(SystemExit):  # Not updated the setup file so error
```
Comment (t-young31) [inline tests/test_no_todo.py]:
```suggestion
        with pytest.raises(SystemExit):  # "TODO" exists
```

File: tests/test_no_todo.py [hunk 2]
```diff
@@ -13,12 +13,51 @@ def test_no_todo(monkeypatch: pytest.MonkeyPatch) -> None:
...
+
+    with TemporaryDirectory() as tmp_dir:
+        os.chdir(tmp_dir)
 
         with open("tmp.py", "w") as file:
-            print("print('hello world')", file=file)
+            print(
+                """
+                todo = 1
+                done = 10 / todo
```
Comment (t-young31) [inline tests/test_no_todo.py:32]: nice catch – thanks 😄

File: tests/test_no_todo.py [hunk 3]
```diff
@@ -13,12 +13,51 @@ def test_no_todo(monkeypatch: pytest.MonkeyPatch) -> None:
...
+
+def test_empty_comment(monkeypatch: pytest.MonkeyPatch) -> None:
+    """Checks that empty comments do not get flagged"""
+    monkeypatch.setattr("sys.argv", ["no_todos.py", "tmp.py"])
+
+    with TemporaryDirectory() as tmp_dir:
+        os.chdir(tmp_dir)
+
+        with open("tmp.py", "w") as file:
+            print("print('hello world') #", file=file)
```
Comment (t-young31) [inline tests/test_no_todo.py:48]: intriguing https://regex101.com/ in python mode is happy with this 🤷🏼

**Review by t-young31 — APPROVED** [review]

File: tests/test_no_todo.py (hunk 3 above)
Comment (gsvarovsky) [reply tests/test_no_todo.py:48]: ah, no, this test arose because of a mistake in my initial correction :)

Comment (t-young31) [reply tests/test_no_todo.py:48]: lovely. more tests = better

**Pull request merged**